### ISSUE TITLE
Add question: "A disabled button fails minimum contrast ratio - is this a fail?

### ISSUE THREAD

**Issue by MaximPerry** [cenw-wscoe/WCAG-in-Plain-Language#22]

**Comment by MaximPerry** [comment]
Added question with Branch Issue22. Closing issue.
